deploy-os_servers.yml

```diff
@@ -434,11 +434,69 @@
         wait: true
         timeout: "{{ openstack_api_timeout }}"
 #############################################################################
-# Configure DAI and SAI from inventory using Openstack API.
+# Configure SAI from inventory using Openstack API.
 #############################################################################
-- name: Create Sys and Deploy Admin Interfaces.
+- name: Create and Deploy SAI Interfaces.
```

Comment:
It is just the name, so not critical but the **_I_** in **_SAI_** stands for Interface: **_S_**ys **_A_**dmin **_I_**nterface, so SAI Interfaces is redundant.

Reply:
will fix right away